astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1557,6 +1567,7 @@ def __init__(
         if parent:
             frame = parent.frame(future=True)
             frame.set_local(name, self)
+        self.doc_node = doc_node
```

Comment:
With the other nodes, the assignments (from arguments) are usually above the `super().__init__(...)` call.